core/constants.go

```diff
@@ -332,13 +332,17 @@ const MetaChainSystemSCsCost = "MetaChainSystemSCsCost"
 // TransactionStatus is the type used to represent the status of a transaction
 type TransactionStatus string
 
+func (ts TransactionStatus) String() string {
```

Comment:
Perhaps remove this if not strictly necessary?

Reply:
Agree about removal. The cast can be easily made where need

Reply:
done method removed.